pkg/scheduler/schedule_one.go

```diff
@@ -655,7 +653,10 @@ func (sched *Scheduler) findNodesThatPassFilters(
 
 	// Stops searching for more nodes once the configured number of feasible nodes
 	// are found.
-	fwk.Parallelizer().Until(ctx, numAllNodes, checkNode, metrics.Filter)
+	if err := fwk.Parallelizer().Until(ctx, numAllNodes, checkNode, metrics.Filter); err != nil {
+		statusCode = framework.Error
+		return feasibleNodes, err
```

Comment:
Do we still need L660-L667 before return?

Reply:
Thanks, I found that feasibleNodes should not be returned here. Fixed it.